### PR TITLE
[IMP] l10n_pe: enhance tax group and taxes for free billing support

### PR DESCRIPTION
**Pull request by desdelinux**
This commit enhances the tax group template in `account.tax.group.pe.csv` and he tax template in `account.tax.pe.csv` to support the free billin functionality by adding a dummy tax group. The related logic is implemented in the enterprise module.

Related Enterprise PR: https://github.com/odoo/enterprise/pull/56767

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_pe/data/template/account.tax.group-pe.csv [hunk 1]
```diff
@@ -11,4 +11,5 @@
 "tax_group_other","OTHERS","0","base.pe","OTROS"
 "tax_group_det","DET","100","base.pe","DET"
 "tax_group_icbper","ICBPER","0","base.pe",""
-"tax_group_ret","RET","100","base.pe","RET"
\ No newline at end of file
+"tax_group_ret","RET","100","base.pe","RET"
+"tax_group_free_invoice","Free Invoice","200","base.pe","Free Invoice"
```
Comment (ren-odoo) [inline addons/l10n_pe/data/template/account.tax.group-pe.csv:15]: @desdelinux just small change pls: the last column is the Spanish translation, can you update it pls with "Factura Gratuita" instead pls.

Comment (desdelinux) [reply addons/l10n_pe/data/template/account.tax.group-pe.csv]: done

**Review by svfu-odoo — CHANGES_REQUESTED** [review]
It is basically good already. :+1:
I left some small comments and a functional question.

File: addons/l10n_pe/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
     'name': 'Peru - Accounting',
     'icon': '/account/static/description/l10n.png',
     'countries': ['pe'],
-    'version': '3.0',
```
[the remark this answers is not in the document]
Comment (desdelinux) [reply addons/l10n_pe/__manifest__.py]: done

File: addons/l10n_pe/data/template/account.tax-pe.csv [hunk 3]
```diff
@@ -1,61 +1,74 @@
...
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"purchase_tax_gra","0%","","GRA","9996","E","0.0","purchase","1","","tax_group_gra","","base","invoice","","","0% Gratis"
-"","","","","","","","","","","","","tax","invoice","chart40111","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"sale_tax_exp","0% EXP","","EXP","9995","S","0.0","sale","1","1","tax_group_exp","","base","invoice","","0","0% EXP"
-"","","","","","","","","","","","","tax","invoice","chart40111","0",""
-"","","","","","","","","","","","","base","refund","","0",""
-"","","","","","","","","","","","","tax","refund","chart40111","0",""
+"id","name","description","invoice_label","l10n_pe_edi_tax_code","l10n_pe_edi_unece_category","amount","amount_type","children_tax_ids","type_tax_use","sequence","include_base_amount","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description"@es
```
[the remark this answers is not in the document]
Comment (desdelinux) [reply addons/l10n_pe/data/template/account.tax-pe.csv]: done

File: addons/l10n_pe/data/template/account.tax-pe.csv [hunk 4]
```diff
@@ -1,61 +1,74 @@
...
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"purchase_tax_gra","0%","","GRA","9996","E","0.0","purchase","1","","tax_group_gra","","base","invoice","","","0% Gratis"
-"","","","","","","","","","","","","tax","invoice","chart40111","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"sale_tax_exp","0% EXP","","EXP","9995","S","0.0","sale","1","1","tax_group_exp","","base","invoice","","0","0% EXP"
-"","","","","","","","","","","","","tax","invoice","chart40111","0",""
-"","","","","","","","","","","","","base","refund","","0",""
-"","","","","","","","","","","","","tax","refund","chart40111","0",""
+"id","name","description","invoice_label","l10n_pe_edi_tax_code","l10n_pe_edi_unece_category","amount","amount_type","children_tax_ids","type_tax_use","sequence","include_base_amount","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@es"
```
Comment (jco-odoo) [inline addons/l10n_pe/data/template/account.tax-pe.csv:1]: Not sure why we have a total diff here instead of only the added taxes...

Comment (svfu-odoo) [reply addons/l10n_pe/data/template/account.tax-pe.csv:1]: The `amount_type` and `children_tax_ids` column had to be added for the group tax.
I (quickly) checked that nothing changed for the existing taxes in the old columns with `csvtool`. And the new column values seemed to be fine too.

**Review by svfu-odoo — APPROVED** [review]
LGTM :+1: 
@jco-odoo for final review

**Pull request closed without merging**